shared/rakefiles/xk_config.rake

```diff
@@ -47,3 +65,16 @@ end
 task :configure_current_project => [@gcp_creds_file] do
   sh "gcloud config set project $TF_VAR_project_id"
 end
+
+task :set_auth_user_vars => [@gcp_creds_file] do
+  # Setting authenticated user's email into env variable, so it can be
+  # accessible in modules: https://issues.gpii.net/browse/GPII-3516
+  ENV['TF_VAR_auth_user_email'] = %x{
+    gcloud auth list --filter='account!~gserviceaccount.com' --format json |  jq -r '.[].account'
```

Comment:
What happens if this command returns more than one thing?

Reply:
This can only happen when more than one user authenticated from the same machine, and rake logic prevents us from doing that.